### PR TITLE
New Module: Seqtk cutN

### LINKED ISSUE #3221: new module: seqtk/cutn
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by DLBPointon**
## PR checklist

Closes #3221  

This PR adds the SeqTK cutN command, this command takes a fasta file and produces a summary bed file detailing the genomic co-ords of lengths of N.

- [X] This comment contains a description of changes (with reason).
- [X] If you've fixed a bug or added code that should be tested, add tests!
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [X] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [X] Add a resource `label`
- [X] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [X] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/seqtk/cutn/meta.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      tool_dev_url: https://github.com/lh3/seqtk
+      licence: ["MIT"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
```
Comment (priyanka-surana) [inline modules/nf-core/seqtk/cutn/meta.yml:20]: Change `reads` to `fasta` to match the `main.nf`
```suggestion
  - fasta:
```

File: modules/nf-core/seqtk/cutn/meta.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      description: A single fasta file to be split.
+      pattern: "*.{fasta}"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
```
Comment (priyanka-surana) [inline modules/nf-core/seqtk/cutn/meta.yml:31]: Change `reads` to `fasta` to match the `main.nf`
```suggestion
  - bed:
```

**Review by priyanka-surana — CHANGES_REQUESTED** [review]
Looks good. A couple changes to `meta.yml`

**Review by nvnieuwk — APPROVED** [review]
One small nitpicky comment but otherwise LGTM!

**Review by priyanka-surana — APPROVED** [review]

**Pull request merged**